js/script.js

```diff
@@ -0,0 +1,253 @@
+        //get the birthday 
+        //user selected month (from dropdown menu) value from 01 - 12
+        var inputMonth = $("#input-month").val();               
+        //user selected day (from dropdown menu)
+        var inputDay = $("#input-day").val();
+        //user input year (typed)
+        var inputYear = $("#input-year").val();
+
+        //make a variable to hold our card image or headlines
+        var cardImage = $("<img>");
+        
+        //make api keys and get functions for each api 
+        //nasa API Function
+        function getNasaImage () {
+            //make query url
+            var nasaURL = "https://api.nasa.gov/planetary/apod?api_key=PxXYhbxjF4qoZihVRPPnN3C4IpvxKF4N1eWiJVRx&date=" +
+                //last year from moment.js
+                lastYear + "-" +
+                //user selected month and day
+                inputMonth + "-" +
+                inputDay;
+            
+            //ajax function using our nasa URL
+            $.ajax({
+                method:"GET",  
+                url: nasaURL,
+            }).then(function(response){
+                //add an src of the nasa high res imagery of the day
+                cardImage.attr("src", response.hdurl);
+                //add an alt
+                cardImage.attr("alt", "space poster image");
+                //apend it to displayed card
+                $(".api-image").append(cardImage);
+            })
+        }
+        //NYT Book API Function
+        function getBookImage() {
+            //make query url
+            var bookURL = "https://api.nytimes.com/svc/books/v3/lists/" +
+                //last year from moment.js
+                lastYear + "-" +
+                //user selected month and day
+                inputMonth + "-" +
+                inputDay +
+                "/hardcover-fiction.json?api-key=PRpWMMh6gJsugEdDfDApTNqb7N7LiRkN";
+            
+            //ajax function using our book URL
+            $.ajax({
+                method:"GET",  
+                url: bookURL,
+            }).then(function(response){
+                //this variable holds a random number between 0 and 9
+                var x = Math.floor((Math.random() * 10));
+                //make a variable holding the bool result at the value of x
+                var bookCover = response.results.books[x].book_image;
+                //add an src of the book cover image 
+                cardImage.attr("src", bookCover);
+                //add an alt
+                cardImage.attr("alt", "book cover image");
+                //apend it to displayed card
+                $(".api-image").append(cardImage);
+            })
+        }
+        //TMDB API Function
+        function getMovieImage() {
+            //make query URL
+            var movieURL = "https://api.themoviedb.org/3/search/movie?api_key=041542fa1899303e8c187b3a599c9d43&language=en-US&query=e&include_adult=false&primary_release_year=" + 
```

Comment:
you want to break up the url a little bit since this line is so long, having to scroll to the right to read is a pain